templates/environment/partials/environment-manager-role.yml

```diff
@@ -203,6 +199,12 @@ EnvironmentManagerRole:
             "tag:GetResources"
           ]
           Resource: "*"
+        - Sid: ApplicationAutoscaling
+          Effect: Allow
+          Action: [
+            "application-autoscaling:DescribeScalingPolicies"
+          ]
+          Resource: "*"
```

Comment:
We might not be able to scope this policy down because of 

> Application Auto Scaling does not support specifying a resource ARN in the Resource element of an IAM policy statement. To allow access to Application Auto Scaling, specify “Resource”: “*” in your policy.

See [here](https://docs.aws.amazon.com/IAM/latest/UserGuide/list_applicationautoscaling.html).